raiden/blockchain/net_contract.py

```diff
@@ -368,7 +368,7 @@ def close(self, ctx, first_encoded, second_encoded):
         partner_state = self.participants[self.partner(ctx['msg.sender'])]
 
         # may be None, a node is not required to make a transfer
-        closer, partner = self._decode(ctx['msg.sender'], first_encoded, second_encoded)
+        closer, partner = self._decode(ctx['msg.sender'], first_encoded, None)
```

Comment:
This method call can be replaced by the function `decode_transfer`, and at first glance it seems the method can be removed.